### PR TITLE
Creating sign in pages

### PR DESCRIPTION
**Pull request by jaimegarjr**

### CODE REVIEW

File: lib/widgets/metric_card_section.dart [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  late Future<Record> record;
+  Future<Record> fetchMetricCardData() async {
+    time = DateTime.now();
+    Map<String, dynamic> data = await NetworkHelper.getData(
+      httpClient,
+      Uri(
+          scheme: 'https',
+          host: 'solaris-web-server.herokuapp.com',
+          path: 'records/most_recent/1'),
+    ); // 'https://solaris-web-server.herokuapp.com'
```
Comment (Oziel-Sal) [inline lib/widgets/metric_card_section.dart:29]: Remove comment

File: lib/widgets/metric_line_chart_section.dart [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  }
+
+  @override
+  void dispose() {
+    _timer.cancel();
+    super.dispose();
+  }
+
+  Future<MetricLineChart> fetchLineChartData() async {
+    // Map<String, dynamic> lineChartData =
```
Comment (Oziel-Sal) [inline lib/widgets/metric_line_chart_section.dart:41]: remove comments

File: lib/widgets/power_bar_chart_section.dart [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  @override
+  void initState() {
+    super.initState();
+    futureBarChart = fetchBarChartData();
+  }
+
+  Future<MetricBarChart> fetchBarChartData() async {
+    Map<String, dynamic> barChartData =
+        json.decode(await getLocalBarChartJson());
+    // Map<String, dynamic> barChartData = await NetworkHelper.getData(
```
Comment (Oziel-Sal) [inline lib/widgets/power_bar_chart_section.dart:29]: comments

**Review by Oziel-Sal — CHANGES_REQUESTED** [review]
Everything looks great. You left some commented code in some of your files.

File: lib/widgets/power_bar_chart_section.dart (hunk 3 above)
Comment (jaimegarjr) [reply lib/widgets/power_bar_chart_section.dart:29]: needed before we set up 7 days of data

**Review by Oziel-Sal — APPROVED** [review]
LGTM

**Pull request merged**